engine/jsonlint/reporter.js

```diff
@@ -0,0 +1,79 @@
+const template = module.exports = {
+    run: function(onFinish) {
+        process.stdin.resume();
+        process.stdin.setEncoding("utf8");
+
+        let lingeringLine = "";
+        let lines = [];
+        
+        process.stdin.on("data", function(chunk) {
```

Comment:
I suggest do not mix different versions of ECMAScript syntax, e.g. use arrow functions.